pmoves/scripts/audit_network_reality.sh

```
@@ -0,0 +1,192 @@
+warn() { echo -e "  ${YLW}⚠${RST}  $*"; WARNINGS=$((WARNINGS+1)); }
+dim()  { echo -e "${DIM}     $*${RST}"; }
+
+DRIFT=0
+WARNINGS=0
+PORTS_ONLY=0
+JSON=0
+
+for arg in "$@"; do
+  case "$arg" in
+    --ports-only) PORTS_ONLY=1 ;;
+    --json)       JSON=1 ;;
```

Shellcheck (0.11.0)

[warning] JSON appears unused. Verify use (or export if used externally).

(SC2034)